api_spec.yaml

```diff
@@ -3736,6 +4247,119 @@ paths:
         '500':
           $ref: '#/components/responses/500'
 
+  /recipe/multitenancy/tenant/user:
+    post:
+      tags:
+        - Multitenancy Recipe
+      description: |
+        Add user's association with the tenant.
+
```

Comment:
add to the description that the userId will be added to the tenant id based on the prefix of the url

Reply:
Done